components/Counter.tsx

```diff
@@ -30,6 +30,9 @@ export default function Counter({
   value,
   onNewValue,
 }: CounterProps): JSX.Element {
+  const ref = React.useRef<any>();
```

Comment:
```suggestion
  const ref = React.useRef<HTMLSpanElement>(null);
```

@typescript-eslint/no-explicit-any 💯